CHANGES

```diff
@@ -12,6 +12,7 @@
 - Android: Add configuration option to render under the cutout if available on the device.
 - Fix: Keep SelectBox popup from extending past right edge of stage.
 - Added Framebuffer multisample support (see GL31FrameBufferMultisampleTest.java for basic usage)
+- Fix: Fonts generated with gdx-ttf no longer bleed when drawn with a shadow
```

Comment:
I think it's called gdx-freetype, but I don't use it much so it might be called gdx-ttf in some places.